pgctl/cli.py

```diff
@@ -15,10 +17,12 @@
 from py._path.local import LocalPath as Path
 
 from .config import Config
+from pgctl.service import Service
 
 PGCTL_DEFAULTS = frozendict({
     'pgdir': 'playground',
     'pgconf': 'conf.yaml',
+    'pghome': Path(os.environ.get('XDG_RUNTIME_DIR') or '~/.run', expanduser=True).join('pgctl').strpath,
```

Comment:
The reason I left it there is that otherwise you get a crazy printed thing like:
`/nail/tmp/pytest-662/it_can_show_its_configuration_2/~/.run/pgctl`

I can change it if you want, presumably we'll probably have a better config printer that could do something more reasonable in the future,